user-interface/src/case-assignment/AssignAttorneyModal.tsx

```diff
@@ -104,17 +103,26 @@ function AssignAttorneyModalComponent(
     };
   });
 
-  function areArraysSame(ar1: string[], ar2: string[]): boolean {
-    return JSON.stringify(ar1.sort()) === JSON.stringify(ar2.sort());
+  function areArraysSame(ar1: AttorneyUser[], ar2: AttorneyUser[]): boolean {
```

Comment:
did we find another way to compare these?

Reply:
YES!!! we can use the new deepEqual method if these are no longer arrays of strings.